my_main.py

```diff
@@ -0,0 +1,67 @@
+# HMMMM what should i do
+# I think book-reading should be an easy one. Or a math computation of some kind.
+# Inputs: number
+# Outputs: all possible ways of adding digits together, printed out
+
+def print_all(n):
+    s = str(n)
```

Comment:
what does this function do? add a doc string to describe for future generations!